docs/source/tutorial/typesfuns.rst

```diff
@@ -1127,11 +1125,11 @@ Nested record fields can be accessed using the dot notation:
     x.a.b.c
     map (.a.b.c) xs
 
-For the dot notation, there must be no spaces after the dots but there may be
-spaces before the dots. The composite projection must be parenthesised,
-otherwise ``map .a.b.c xs`` would be understood as ``map.a.b.c xs``.
+For the dot notation, there must be no spaces after the dots, but there may be
```

Comment:
In other instances of `x`-notation, a hyphen has been inserted. We should be consistent within the documentation.